### ISSUE TITLE
Alcohol/Ethanol energy density is wrong

### ISSUE THREAD

**Issue by anoobindisguise** [CleverRaven/Cataclysm-DDA#60818]
### Is your feature request related to a problem? Please describe.

Alcohol fuels have the wrong energy density. Per https://www.iea-amf.org/content/fuel_information/fuel_info_home/ethanol/e10/ethanol_properties/ I get about 27mJ of ethanol per kg, or 21 mJ per liter. So that's 27kJ/gram, 21kJ/mL.
However, in game ethanol, methanol, denatured alcohol, etc have a density of 1:1, they all weight 1g per 1mL. Thus they get differing energy densities.
However, in game Alcohol (the fuel used for ethanol, denatured alcohol, etc) are all 15.6kJ per mL. This is vastly too low, it should be at about 33% higher. Denatured alcohol is usually within 98% of ethanol energy IIRC. 

### Solution you would like.

Can weight be specified to less than 0.001kJ per charge? If so that should be done, and then energy density should be corrected in addition to that per the alcohol material. As far as I know alcohol material is used as fuels basically only for concentrated alcohol, and for vodka and other hard liquors for the ethanol cell bionic. 

### Describe alternatives you have considered.

Split alcohol into several different types to represent the varying proofs?

**Comment by borsek** [comment]
I've been thinking about revamping the chemistry system, how to tackle all of it, whether to go with actual densities of fluids, trivial or chemical names with concentration percentages, energy, reaction yield, etc.

- First of all, you mean MJ, not mJ, capitalization is important, since m = mili, 10^-3, M = Mega, 10^+6.
- Second, I think denaturated alcohol is redundant flavor. Ethanol is supposedly 100% ethanol. Denaturated alcohol I am assuming is the azeotropic mixture at 95% ethanol 5% water, as is the case in hardware store bottles.
We can make ethanol from distilling alcohol, with no drying agent, instead of making denaturated alcohol, then using a drying agent to get ethanol. So this is already WTF.
- Third, a bunch of liquids in game have the density of water, it seems - even oils tend to be 1.0 instead of 0.8-ish g/ml. They also have the melting point of water - ethanol doesn't freeze until -114 °C, yet iirc it freezes, much like antiseptic and a bunch of other liquids in game, at 0 °C.
- Fourth, and this is a clarity and coding problem, fluids have varying base units. Most (but not all) chemicals go by 1ml, most food and water go by 250ml as (1) unit. These can often be mixed to get varying outputs.

So with all these errors, as well as the wildly inaccurate energy consumptions of some items - 10W night vision vs. what feels like 0.1W thermal vision, etc. I don't think highly accurate energy density is worth worrying about and going to 0.1J per charge accuracy for.

After all this is a game, and going with IRL values gets tedious at a certain point. How far are you actually willing to go, and how much is even feasible for simulation (due to the eventual laaaaag). Much better to have 'whole values', and relative rough relations between them - otherwise, are we going to assign a measly 30% energy efficiency of a thermal combustion engine to the burning process? What about the bionics, do they use combustion or fuel cells, which have much better yields?

TL;DR: everything is so wrong with energy density that a slight difference in it between practically identical fluids is the least of our worries, imho.
